MoEPluginInbox/MoEInboxUtils.swift

```diff
@@ -0,0 +1,117 @@
+//
+//  MoEInboxUtils.swift
+//  MoEPluginInbox
+//
+//  Created by Rakshitha on 11/07/22.
+//
+
+import Foundation
+import MoEngageInbox
+import MoEPluginBase
+
+class MoEInboxUtils {
```

Comment:
@RakshithaAcharya Let's see if we can make use of protocols here as well

Reply:
@deepa-iOS-SDK  done